frontend/public/plugins.ts

```diff
@@ -30,13 +25,29 @@ export const initConsolePlugins = _.once((reduxStore: Store<RootState>) => {
   registerPluginEntryCallback(pluginStore);
 });
 
+const loadPluginFromURL = async (baseURL: string) => {
+  const manifest = await fetchPluginManifest(baseURL);
+  loadDynamicPlugin(baseURL, manifest);
+};
+
+// Load all dynamic plugins which are currently enabled on the cluster
+Promise.all(
+  window.SERVER_FLAGS.consolePlugins.map((pluginName) =>
+    loadPluginFromURL(`/api/plugins/${pluginName}`),
```

Comment:
There can technically be a context root we need to take into account, so we should use `window.SERVER_FLAGS.basePath`

```suggestion
    loadPluginFromURL(`${window.SERVER_FLAGS.basePath}api/plugins/${pluginName}`),
```

Reply:
Makes sense, I'll do the code change.